.github/workflows/build.yml

```diff
@@ -250,7 +250,7 @@ jobs:
       run:
         shell: bash -el {0}
     steps:
-      # These names should correspond to SP_INSTALLER_ARTIFACT_ID in tools/extract_version.sh
+      # These names should correspond to PKG_INSTALLER_ARTIFACT_ID in tools/extract_version.sh
       - uses: actions/download-artifact@v4
         with:
           pattern: ${{ env.PROJECT_NAME }}-Python-*
```

Comment:
this line looks off to me, why the trailing `-Python-` before the `*`?

Reply:
Fixed.